Makefile

```diff
@@ -28,6 +28,7 @@ OCI_RUNTIME ?= ""
 
 BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
 ZSHINSTALLDIR=${PREFIX}/share/zsh/site-functions
+FISHINSTALLDIR=${PREFIX}/share/fish/completion
```

Comment:
s/completion/completions/ ... at least on Fedora. Is it a typo or does openSUSE have a different path?

Reply:
Your're right, it should be `completions`.